test/aqua/test_hhl.py

```diff
@@ -42,8 +42,7 @@ class TestHHL(QiskitAquaTestCase):
 
     def setUp(self):
         super(TestHHL, self).setUp()
-        self.random_seed = 0
-        aqua_globals.random_seed = self.random_seed
+        aqua_globals.random_seed = None
```

Comment:
Should this not still be set to 0 instead of None? Or does None default to 0?

Reply:
I would like to leave 0, but tried this and different seeds and got very different results. In particular there was this fidelity that was different than 1. Strangely, None works without any further change, but I would feel more comfortable with a seed, when you pass None: "If None, then fresh, unpredictable entropy will be pulled from the OS" according to numpy docs.

Reply:
It should be initialized to None so we take a random seed from numpy and the behavior is random. Otherwise a non None value will be a fixed seed things and the outcome predictable which is what we often use when testing.